### DISCUSSION TITLE
Having an issue entering password

### DISCUSSION THREAD

**Discussion by NateDawg7** [community/community#43090] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

On the last step when trying to enter my password it just shows a key
Username for 'https://github.com': "My username"
Password for 'https://NateDawg7@github.com': "Key" which doesn't allow any input, tried multiple different terminals and still no use.

**Comment by airtower-luna** [comment]
Two things:

1. The password input won't *show* any input (no, not even stars). It'll still accept your input.
2. You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
